eo-maven-plugin/src/main/java/org/eolang/maven/DcsFile.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016-2022 Objectionary.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.maven;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.apache.maven.model.Dependency;
+
+/**
+ * Plugin saves all transitive dependencies to a file.
+ *
+ * @since 0.28.11
+ */
+interface DcsFile {
```

Comment:
@volodya-lombrozo Is this entity actually related to file? What property of its contract tell us something about the file? As I see from its methods it something with dependencies.
Generally it's not clear what this entity represents.

Reply:
@mximp Fixed. The class and his method was renamed.

Reply:
@mximp 
```java
interface DependenciesFile {
    Path file(Dependency origin);
}
```

The idea is return the file which contains all transient dependencies of the `origin` dependency.